Job/JobParameters/DefaultValuesProvider/FamilyCsvExport.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Pim\Bundle\EnhancedConnectorBundle\Job\JobParameters\DefaultValuesProvider;
+
+use Akeneo\Component\Batch\Job\JobInterface;
+use Akeneo\Component\Batch\Job\JobParameters\DefaultValuesProviderInterface;
+
+/**
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class FamilyCsvExport implements DefaultValuesProviderInterface
```

Comment:
Let's discuss about this naming tomorrow!
My concern is that having 2 FamilyCsvExport classes could induce an error has it is not implicit that one is a `DefaultValuesProviderInterface` and the other one a `ConstraintCollectionProviderInterface`.